content/Google is a modal dialog - Martin Rubi.md

```diff
@@ -2,101 +2,101 @@
 
 ### The shame of being a modal dialog
 
-A modal dialog is a window which pops up on top of every other window freezing the
-underlaying user interface until the user dismisses it.
+Modal dialogs are windows that pops-up on top of every other window freezing the
```

Comment:
if you are using plural then you should use `pop-up`

Reply:
Fixed it. Thanks!